Source/WebCore/loader/archive/ArchiveResource.h

```diff
@@ -36,22 +36,25 @@ class ArchiveResource : public SubstituteResource {
 public:
     static RefPtr<ArchiveResource> create(RefPtr<FragmentedSharedBuffer>&&, const URL&, const ResourceResponse&);
     WEBCORE_EXPORT static RefPtr<ArchiveResource> create(RefPtr<FragmentedSharedBuffer>&&, const URL&,
-        const String& mimeType, const String& textEncoding, const String& frameName,
-        const ResourceResponse& = ResourceResponse());
+    const String& mimeType, const String& textEncoding, const String& frameName, const ResourceResponse& = ResourceResponse(), const String& fileName = { });
```

Comment:
Nit: The arguments should be all on one line, or they should be indented.  IMO, it's hard to read when arguments continue on a second line, but also line up with new functions.

If indenting triggers a `check-webkit-style` warning, we should fix that instead (but we do not have to fix that with this PR).